### PR TITLE
Merge works and about sections into main

### PR DESCRIPTION
**Pull request by khitermedachraf**
I added the HTML and the CSS of the works and the about me section.
I used the Grid layout for all the card containers to create a grid column with auto-fit
I used the flexbox layout for all the card placeholders, flex-column for the placeholder, and also flex-row for skills and languages tag.
I fixed all Linters errors added and fixed some PNGs to the background

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,6 +54,147 @@ <h1 class="light-green">I’m a software developer</h1>
...
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
```
Comment (rukundoeric) [inline index.html]: - [x] <b>REQUIRED:</b> As mentioned in the project requirements, each project card should have an image that uses the tag `img` and a description that uses paragraph `p`.
Screen Shot 2022-03-19 at 19 59 09

File: index.html [hunk 2]
```diff
@@ -51,6 +54,147 @@ <h1 class="light-green">I’m a software developer</h1>
...
+        </div>
+      </section>
+      <section id="about-me">
+        <img class="aboutme-img1" src="img/aboutme-img1.png" alt="png1" />
```
Comment (rukundoeric) [inline index.html]: - [x] <b>REQUIRED:</b> The project requires to use background images, not img tags. Kindly fix it.
scrnli_3_19_2022_8-03-35 PM

File: index.html [hunk 3]
```diff
@@ -51,6 +54,147 @@ <h1 class="light-green">I’m a software developer</h1>
...
+        <h2 class="dark-blue">About me</h2>
+        <p class="dark-blue">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <img class="aboutme-img2" src="img/aboutme-img2.png" alt="png2" />
```
Comment (rukundoeric) [inline index.html]: - [x] <b>REQUIRED:</b> Same thing goes here. The project requires to use background images, not img tags. Kindly fix it.
scrnli_3_19_2022_8-03-35 PM

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @khitermedachraf, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (khitermedachraf) [reply index.html]: Thank you so much @rukundoeric for your observations and review, but I didn't find these requirements in the project description.
also, I used template 03 which doesn't include any paragraphs to add.
Do requirements change from one template to another?

File: index.html (hunk 3 above)
Comment (khitermedachraf) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (khitermedachraf) [reply index.html]: Done

**Review by Deepakdanger — APPROVED** [review]
Hi @khitermedachraf, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] It is nice to have the button hover, pressed design in the project, This is recommended changes kindly change this before merging. 
- Screenshot from 2022-03-20 02-06-21

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by khitermedachraf** [comment]
Okey no problem, I will change it.

**Comment by khitermedachraf** [comment]
All suggestions are implemented perfectly.
yeah it's time to merge.

**Pull request merged**